docs/src/app/pages/codibotintro.md

```diff
@@ -0,0 +1,35 @@
+# Platform: MIT App Inventor Codi Bot
+
+The MIT App Inventor Codi Bot is a hands-on IoT kit. Our adorable mascot can be controlled via App Inventor through Bluetooth communication. This educational kit is easy to use and guides users through the whole process of development, from building a robot to programming it. 
+
+<div style="text-align: center; font-size: 75%; margin: 16pt 0;">
+![Image of the MIT App Inventor Codi Bot.](/assets/howtos/codibot1.jpg)
+
+</div>
+
+## Parts list (<a href="https://www.robotkingdom.com.tw/product/mit-app-inventor-codi-bot/" target="_blank">Purchase link)</a>:
+* <a href="https://www.robotkingdom.com.tw/product/linkit-7697/" target="_blank">LinkIt 7697 MCU board</a>, 1
+* <a href="https://www.robotkingdom.com.tw/product/linkit-7697%E6%A9%9F%E5%99%A8%E4%BA%BA%E6%93%B4%E5%85%85%E6%9D%BF-robot-shield-for-linkit-7697/" target="_blank">Robot Shield for LinkIt 7697 V2.0</a>, 1 (with an onboard buzzer) 
+* RGB LED (common cathode), 1
```

Comment:
Are we intending to link these other components? It seems odd to me that only the first two are linked.

Reply:
Cuz the first two are the core component of Codi Bot, other are quite common and easy to get. You suggest I link all of them or remove the link of first two?